WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -444,12 +450,20 @@ extension MainTabBarController {
 
     private static func presentDetails(for orderID: Int64, siteID: Int64) {
         if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
-            (childViewController() as? OrdersSplitViewWrapperController)?.presentDetails(for: orderID, siteID: siteID)
+            ordersTabSplitViewWrapper()?.presentDetails(for: orderID, siteID: siteID)
```

Comment:
As part of a fix for one/both of the known issues, we could have `OrdersSplitViewWrapperController.presentDetails(for:)` delegate to the existing method with the same name on `OrdersRootViewController`. 

That way, we:
1. remove some duplication
2. can co-ordinate changes to the list and the presented order simultaneously.